examples/ha/send_alerts.sh

```diff
@@ -1,3 +1,4 @@
+#!/bin/bash
```

Comment:
can you replace this with `#!/usr/bin/env bash`?

Reply:
updated